.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,18 @@
-**Description of work**
+**Description of work.**
+
+**Report to:** [user name]/[nobody]. <!--If the original issue was raised by a user they should be named here.-->
 
 **To test:**
 
 <!-- Instructions for testing. -->
 
 Fixes #xxxx. <!-- and fix #xxxx or close #xxxx xor resolves #xxxx -->
 
-**Release Notes** 
+Does this update require release notes?
+- [ ] Yes
+- [ ] No
 <!--
-Either edit the file in docs/source/release/... and it will be in your pull request or state
-*Does not need to be in the release notes.*
+If yes, edit the file docs/source/release/... or state release notes.
```

Comment:
I think `If yes, edit the file docs/source/release/...` is sufficient, I am not sure what `state release notes` means? The or used to be state that it does not need to be in the release notes, which is now covered.

Reply:
Ah my mistake - it was unclear what the or clause of the previous comment meant, and I misinterpreted.